var/spack/repos/builtin/packages/wireshark/package.py

```diff
@@ -65,7 +70,14 @@ def cmake_args(self):
         args = ['-DENEABLE_CARES=ON',
                 '-DENABLE_GNUTLS=ON',
                 '-DENABLE_LUA=ON',
-                '-DENABLE_MAXMINDDB=ON']
+                '-DENABLE_MAXMINDDB=ON',
+                '-DYACC_EXECUTABLE=' + self.spec['bison'].prefix.bin.yacc,
+                '-DGIT_EXECUTABLE=' + self.spec['git'].prefix.bin.git,
+                '-DPCAP_INCLUDE_DIR=' + self.spec['libpcap'].prefix.include,
+                '-DPCAP_LIB=' + self.spec['libpcap'].prefix.libs,
```

Comment:
This won't work.

`self.spec['libpcap'].prefix` is an object of type `Prefix`. It's basically a class that stores filepaths, similar to the `pathlib` library without most of the features. Basically, it lets you specify subdirectories by using Python attributes. So `spec['libpcap'].prefix.libs` will expand to the `<prefix>/libs` directory, which probably doesn't exist.

You want to use `spec['libpcap'].libs`, which is an object of type `LibraryList`. This class allows you to manipulate libraries, requesting the library names, directories, and linking flags.

Same thing below, you want `self.spec['lua'].libs`.

Reply:
> I completely don't understand how the `.libs` property is created

`LibraryList` and the methods that create them are defined in `lib/spack/llnl/util/filesystem.py`. The default libs handler is defined in `lib/spack/spack/spec.py`. There are many different options you can use if you expect to find multiple libraries but only want a single one. Let me know if you have any other questions.